### DISCUSSION TITLE
Queue count

### DISCUSSION THREAD

**Discussion by Skuwwy** [Comfy-Org/ComfyUI#53] [Q&A]
Hello, I just switched to comfy from A1111. And I'm loving it!

Is there a way I can set the queue number to something like 50 and let it run for a while? My system cannot handle batches any higher than 1.

Editing this post to ask another question : Is there a way to add a button to browse for controlnet input image instead of placing them in the folder beforehand?

**Comment by comfyanonymous** [comment]
>Is there a way to add a button to browse for controlnet input image instead of placing them in the folder beforehand?

That's implemented now.
